devicon.json

```
@@ -4104,6 +4104,24 @@
         "color": "#3C99D4",
         "aliases": []
     },
+    {
+        "name": "webassembly",
+        "tags": [
+            "web-environment"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "original",
+                "plain"
+            ]
+        },
+        "color": "#654FF0",
+        "aliases": []
```

I suggest removing the `webassembly-plain.svg` file and adding the `plain` version as an alias.
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
        ]
```